src/components/MoviesList/MoviesList.tsx

```diff
@@ -1,6 +1,14 @@
 import React from 'react';
 import './MoviesList.scss';
+import { Movie } from '../../types/Movie';
+import { MovieCard } from '../MovieCard';
 
-export const MoviesList: React.FC = () => (
-  <>Put the list here</>
+type MoviesProps = {
+  movies: Movie[];
+};
+
+export const MoviesList: React.FC<MoviesProps> = ({ movies }) => (
+  <>
+    {movies.map((movie: Movie) => <MovieCard movie={movie} />) }
+  </>
```

Comment:
Use semantic tags for list and it's items
And you should use key attribute when rendering array